api/src/opentrons/protocol_engine/execution/movement.py

```diff
@@ -109,15 +108,12 @@ async def move_to_well(
         dest_slot_int = int(
             self._state_store.geometry.get_ancestor_slot_name(labware_id)
         )
-        hw_pipette = self._state_store.pipettes.get_hardware_pipette(
-            pipette_id=pipette_id,
-            attached_pipettes=self._hardware_api.attached_instruments,
-        )
 
         self._hs_movement_flagger.raise_if_movement_restricted(
             hs_movement_restrictors=hs_movement_restrictors,
             destination_slot=dest_slot_int,
-            is_multi_channel=hw_pipette.config["channels"] > 1,
+            is_multi_channel=self._state_store.tips.get_pipette_channels(pipette_id)
+            > 1,
```

Comment:
Suggestion for readability. Alternatively, add a selector to the TipsView that can handle the `>` logic

```suggestion
            is_multi_channel=(
                self._state_store.tips.get_pipette_channels(pipette_id) > 1
            ),
```